pwndbg/commands/argv.py

```diff
@@ -132,8 +132,11 @@ class environ_function(gdb.Function):
     def __init__(self) -> None:
         super().__init__("environ")
 
-    def invoke(self, name):
-        name = name.string() + "="
+    def invoke(self, name_value: gdb.Value = gdb.Value(""), *args: gdb.Value) -> gdb.Value:
```

Comment:
Tbh I didn't even know we had such funcs. Is this sth like `print $environ("sth")` and `print $argv(0)` etc? nice